bin/node-template-archive/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "node-template-archive"
 version = "0.3.0"
 authors = ["Andrew Plaza <aplaza@liquidthink.net>"]
```

Comment:
unrelated to this PR: @insipx we should probably as `parity` as authors too to all crates?!

Reply:
happy to use whatever, you feel free to commit in this PR's branch too whatever you want. perhaps: 
 https://github.com/paritytech/substrate/blob/master/bin/node/runtime/Cargo.toml#L4
 ```toml
 authors = ["Parity Technologies <admin@parity.io>"]
 ```

Reply:
updated the last commit to use this throughout, please `git reset HEAD~ --hard` if you think it's not appropriate in this PR (new title)